ServiceLoader osgi problems

There is a problem with class AbstractCalendarValidatorFactory similar to #125, running code in an osgi system.

> java.lang.ExceptionInInitializerError
	at net.fortuna.ical4j.model.Calendar.<init>(Calendar.java:159)
	at net.fortuna.ical4j.model.Calendar.<init>(Calendar.java:142)
	at net.fortuna.ical4j.data.CalendarBuilder$ContentHandlerImpl.startCalendar(CalendarBuilder.java:307)
	at net.fortuna.ical4j.data.CalendarParserImpl.parse(CalendarParserImpl.java:119)
	at net.fortuna.ical4j.data.CalendarBuilder.build(CalendarBuilder.java:198)
	at net.fortuna.ical4j.data.CalendarBuilder.build(CalendarBuilder.java:178)
	at net.fortuna.ical4j.data.CalendarBuilder.build(CalendarBuilder.java:165)
	at net.fortuna.ical4j.model.TimeZoneRegistryImpl.loadVTimeZone(TimeZoneRegistryImpl.java:237)
	at net.fortuna.ical4j.model.TimeZoneRegistryImpl.getTimeZone(TimeZoneRegistryImpl.java:205)
	....
Caused by: java.util.NoSuchElementException
	at java.util.ServiceLoader$LazyIterator.nextService(ServiceLoader.java:365)
	at java.util.ServiceLoader$LazyIterator.next(ServiceLoader.java:404)
	at java.util.ServiceLoader$1.next(ServiceLoader.java:480)
	at net.fortuna.ical4j.validate.AbstractCalendarValidatorFactory.<clinit>(AbstractCalendarValidatorFactory.java:12)
	... 41 more

--- a/src/main/java/net/fortuna/ical4j/validate/AbstractCalendarValidatorFactory.java
+++ b/src/main/java/net/fortuna/ical4j/validate/AbstractCalendarValidatorFactory.java
@@ -9,7 +9,7 @@ public abstract class AbstractCalendarValidatorFactory {
 
     private static CalendarValidatorFactory instance;
     static {
-        instance = ServiceLoader.load(CalendarValidatorFactory.class).iterator().next();
+        instance = ServiceLoader.load(CalendarValidatorFactory.class, DefaultCalendarValidatorFactory.class.getClassLoader()).iterator().next();
     }
 
     public static CalendarValidatorFactory getInstance() {

